support-frontend/assets/pages/contributions-landing/contributionsLandingInit.ts

```diff
@@ -261,8 +266,8 @@ function selectInitialContributionTypeAndPaymentMethod(
 	return contributionType;
 }
 
-const init = (store: Store<State, Action, (...args: any[]) => any>) => {
-	const { dispatch } = store;
+const init = (store: Store<State, Action>): void => {
+	const dispatch = store.dispatch as ThunkDispatch<State, void, Action>;
```

Comment:
I'm not sure if there's a better way of doing this. The `store.dispatch` is of type `Dispatch<Action>`, but we want it to be a `ThunkDispatch`. I couldn't easily find a way of getting that into the `Store<State, Action>` type signature